ansible/nimbusserver.yml

```diff
@@ -221,6 +221,10 @@
       regexp: '^(.*)MaxSessions(.*)$'
       replace: 'MaxSessions=1'
       backup: yes
+
+  - name: Remove existing chrome directory
```

Comment:
Isn't this redundant with the change made below for the bookmarks?

Reply:
Indeed, forgot I already added this. I'll remove this one with my next push.